backend/agentd/session.go

```diff
@@ -576,7 +575,7 @@ func (s *Session) stop() {
 }
 
 // handleKeepalive is the keepalive message handler.
-func (s *Session) handleKeepalive(ctx context.Context, payload []byte) error {
```

Comment:
If the context isn't used in this function it can be removed from the argument list, no need to preserve an unexported function's signature.

Reply:
The method actually implements `MessageHandlerFunc` and the context is required in at least one of the implementation.

https://github.com/sensu/sensu-go/blob/8c0a2bd0dc48a9d3f6bfc7aedc1e4569a18c9661/handler/handler.go#L11